source/_drafts/Inband-Captions-Support-with-VHS.md

```diff
@@ -0,0 +1,93 @@
+---
+title: In-band Captions Support with videojs-http-streaming
+author:
+  name: Lahiru Dayananda
+  github: ldayananda
+tags:
+  - vhs
+  - videojs-http-streaming
+  - playback
+  - captions
+  - cea-608
+  - 608
+---
+
+With the release of [videojs-http-streaming](https://blog.videojs.com/introducing-video-js-http-streaming-vhs/) v1.2.0 on July 16th 2018, Video.js has built-in support for CEA/CTA-608 captions carried in FMP4 segments. This means that closed captions are automatically parsed out and made available to Video.js players for MPEG-DASH content and HLS streams using FMP4 segments.
```

Comment:
You can add another paragraph and embed your demuxed talk as a "general overview" thing.

Reply:
```
For a general overview, you can watch [my talk from demuxed](link)
{{embed youtube link via hexo somehow}}
```
Or something

Reply:
https://hexo.io/docs/tag-plugins#YouTube